setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pep8]
 max-line-length=999
 ignore=E121,E123,E126,E226,E24,E704,E221,E127,E128,W503,E731,E131,E711,E712
-exclude=amazon_ses.py,test_*
+exclude=amazon_ses.py
```

Comment:
Since our unit tests are working again, we no longer have our PEP-8 checker ignore them.